app/page.js

```diff
@@ -1,95 +1,38 @@
-import Image from "next/image";
+import Link from "next/link";
 import styles from "./page.module.css";
 
 export default function Home() {
   return (
     <div className={styles.page}>
       <main className={styles.main}>
-        <Image
-          className={styles.logo}
-          src="https://nextjs.org/icons/next.svg"
-          alt="Next.js logo"
-          width={180}
-          height={38}
-          priority
-        />
+        <h1>Advanced Routing</h1>
         <ol>
           <li>
-            Get started by editing <code>app/page.js</code>.
+            <Link href="/render_ssr">
+              NASA Image of the day - Server Side Rendering
+            </Link>
+          </li>
+          <li>
+            <Link href="/rover_photos">
+              Rover Photos - Render using useEffect()
+            </Link>
+          </li>
+          <li>
+            <Link href="/latest_image">EPIC latest image available</Link>
+          </li>
+          <li>
+            <Link href="/nasa_image">EPIC image using dynamic route</Link>
+          </li>
+          <li>
+            <Link href="/nav_bar">Responsive navbar with Material UI</Link>
```

Comment:
What I suggest is that you implement that Navbar and footer globally (ie. in every page -  how would you do that cleanly, without copy pasting in each page component? 🤔) rather than have links to the component.